src/components/organisms/Gallery/Gallery.jsx

```diff
@@ -1,14 +1,13 @@
 import GalleryGrid from "../../molecules/GalleryGrid/GalleryGrid";
 import "./Gallery.scss";
 
-const Gallery =(props)=> {
-
-    return(
-      <div className="gallery">
-        <h1 className="heading">{props.data.gallery.title}</h1>
-        <h2 className="subheading">{props.data.gallery.subTitle}</h2>
-        <GalleryGrid galleryGrid ={props.data.gallery.galleryImages}/>
-      </div>
-    );
-}
-export default Gallery;
\ No newline at end of file
+const Gallery = (props) => {
+  return (
+    <div className="gallery">
+      <h1 className="heading">{props.data.gallery.title}</h1>
```

Comment:
Please send only necessary data to this component, do not send all data of the structure.